packages/blue-api-sdk/src/graphQL.errors.ts

```diff
@@ -0,0 +1,3 @@
+export enum GraphQLErrorCode {
```

Comment:
```suggestion
export enum BlueApiErrorCode {
```

I don't think it is a graphql thing, it more depends on how we expose our graphql API (with what framework, currently nestjs)